src/csf/transformCsf.ts

```diff
@@ -126,7 +127,14 @@ export const transformCsf = (
         acc[key].play = annotations.play;
       }
 
-      acc[key].tags = csf._stories[key].tags || csf.meta?.tags || [];
+      acc[key].tags = combineTags(
+        'test',
+        'dev',
+        ...(csf.meta?.tags || []),
+        ...(csf._stories[key].tags || []),
+        ...previewAnnotations.tags
```

Comment:
This needs to move above the meta for tag precedence

Reply:
Great catch, thanks! Fixed and test written